zerver/worker/queue_processors.py

```diff
@@ -194,8 +192,6 @@ def update_statistics(self, remaining_queue_size: int) -> None:
             update_time=time.time(),
             recent_average_consume_time=recent_average_consume_time,
             current_queue_size=remaining_queue_size,
-            queue_last_emptied_timestamp=self.queue_last_emptied_timestamp,
-            consumed_since_last_emptied=self.consumed_since_last_emptied,
```

Comment:
I think these values are useful to have in the statistics, even if we're not using them to make decisions; they have debugging utility.